README.md

```diff
@@ -8,6 +8,33 @@ Demo: https://www.youtube.com/watch?v=HXdQd65CwkU
 ## Status
 WIP but playable.
 
+## Installation
+
+Prebuilt binary releases are available if you're looking for testing the ruleset or don't possess a developpment environment:
+| [Releases Page](https://github.com/swoolcock/rush/releases/) |
+| -------------|
+
+**NOTE**: For the moment, only desktop platforms (Windows, mac OSX, Linux) are supported.
+
+The ruleset consists of a single DLL file that you'll need to drop in the `rulesets` directory of osu!lazer data directory.
```

Comment:
```suggestion
The ruleset consists of a single DLL file that you'll need to drop it in the `rulesets` directory inside the osu!lazer data directory.
```

Reply:
@Game4all 's is more grammatically correct.